sdk/src/main/java/co/omise/android/models/PaymentMethod.kt

```diff
@@ -0,0 +1,43 @@
+package co.omise.android.models
+
+import android.os.Parcel
+import android.os.Parcelable
+import org.json.JSONObject
+
+class PaymentMethod : Model {
+    var name: String? = null
+    var currencies: List<String>? = null
```

Comment:
Is it possible to have this property and `name` to be NonNull and those the other twos below if it's possible.

Reply:
Unfortunately, not at this moment because then we'll have to manually handle null cases, else it'll crash. Once we put in a serialization library like Jackson, then we can easily make such changes 👍

Reply:
I see. OK then